InsertGenerator/configs/config.ini

```diff
@@ -0,0 +1,2 @@
+[db]
+connection_string: mariadb+mariadbconnector://root:root@127.0.0.1:3306/test4
```

Comment:
This string should be replaced with an env variable, try: 
`connection_string: ${DB_CONNECTION_STRING}`